app/push_notifications/push_notifications.ios.js

```diff
@@ -30,7 +30,8 @@ class PushNotification {
                 ...notification.getData(),
                 message: notification.getMessage(),
             };
-            this.handleNotification(info, false, false);
+            const userInteraction = AppState.currentState === 'active';
```

Comment:
please review and merge this one https://github.com/mattermost/mattermost-mobile/pull/2952